app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -36,11 +37,25 @@ def plan(old_version, environments, options = {})
                                                           :notes => description)
 
           copy_action_outputs = []
+          repos_to_clone = repos_to_copy(old_version, new_components)
 
           sequence do
+            repository_mapping = plan_action(ContentViewVersion::CreateRepos, new_content_view_version, repos_to_clone).repository_mapping
+
+            repos_to_clone.each do |source_repos|
+              plan_action(Repository::CloneToVersion,
+                          source_repos,
+                          new_content_view_version,
+                          repository_mapping[source_repos],
+                          incremental: true)
```

Comment:
i just realized you were dep solving the initial clone_to_version for each repository, we don't actually want to do that right?  we just want to dep solve the things we're adding as part of the incremetnal update?

Reply:
unless i misunderstood?

Reply:
Yes that is unclear, but CloneToVersion uses 'incremental' for a different reason.
If you looked at
https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/clone_to_version.rb#L8-L11
You would see the following line
```filters = incremental ? [] : content_view.filters.applicable(repositories.first)```
That is the only purpose of the 'incremental' flag

Reply:
We are not really dep solving  in CloneToVersion unless the content view has dep solve turned on.